posthog/decorators.py

```diff
@@ -54,7 +54,7 @@ def wrapper(*args, **kwargs) -> Dict[str, Union[List, datetime, bool]]:
                 if filter:
                     dashboard_items = DashboardItem.objects.filter(team_id=team.pk, filters_hash=cache_key)
                     dashboard_items.update(last_refresh=now())
-            return result
+            return {**result, "cache_key": cache_key}
```

Comment:
How is this key being used after it's returned now?

Reply:
I added this more for debugging but can remove it